docs/reference/modules/asset.md

```diff
@@ -0,0 +1,205 @@
+| [`watch`](#watch) | Boolean | If set to `false`, none of the UI assets will be watched for triggering a restart. |
+| `watchDebounceMs` | Integer | Time in milliseconds to wait before re-triggering a restart on asset change. |
+
+### `refreshOnRestart`
+
+If `refreshOnRestart` is truthy and `process.env.NODE_ENV` is not set to `production`, then restart of the Apostrophe app will trigger a browser refresh.
+
+### `watch`
+
+If `watch` is falsy, then changes to any of the assets in the `modules` and `node_modules` will not trigger a restart. Note that this option is disabled and has no impact if `process.env.NODE_ENV` is set to `production`. 
+
+## Command Line Tasks
+
+### `build`
+The build command triggers the compilation, processing, and output of the project assets. The main code from each module, code placed in the `<module-name>/ui/src` folders, and scripts included in [bundles](#bundles) are combined separately. Any SCSS files are combined and translated to CSS. If `NODE_ENV=production`, all of the final deployment files are written out to the `public/releases` folder within a sub-folder named by the release id. The `build` script will try to automatically set the release id when building from a simple git checkout from the hash, building for Heroku or platform.sh from the environment variables, or Stagecoach from the root directory name. Otherwise, the release id can set from a `APOS_RELEASE_ID` environment variable or with a root directory file named 'release-id' with the value of the id inside. The value should be a short, unique string identifier.
+
+#### Example
+
+<AposCodeBlock>
+
+```sh
+APOS_RELEASE_ID='2022-01-01' node app @apostrophecms/asset:build
+```
+</AposCodeBlock>
+
+### `clear-cache`
+Some changes to the webpack configuration will not be detected automatically and will cause incorrect output. While this is rare, the caches for webpack can be emptied from the command line using the `clear-cache` task.
+
+#### Example
+
+<AposCodeBlock>
+
+```sh
+node app @apostrophecms/asset:clear-cache
+```
+</AposCodeBlock>
+
+## Webpack configuration
+
+The webpack configuration can be extended from any module. The `asset` module exposes three different properties for accomplishing this.
+
+## Webpack options
+|  Property | Type | Description |
+|---|---|---|
+| [`extensions`](#extensions) | Object | Adds extra processing functionality to webpack. |
+| [`extensionOptions`](#extensionoptions) | Object | Adds functionality to any extensions added through the `extensions` option. |
+| [`bundles`](#bundles) | Object | |
+
+### `extensions`
+The `extensions` option allows the addition of custom methods for processing the Javascript and SCSS in the `ui/src` folder. The schema used within the option conforms to that used for configuring webpack, which you can learn more about [here](https://webpack.js.org/configuration/). This property can take either a simple object with a name property or a function that returns an object. 
```

Comment:
a function that accepts an object of options and returns an object

Reply:
Changed.